Packs/GuardiCore/Integrations/GuardiCore/GuardiCore.yml

```diff
@@ -49,7 +49,7 @@ script:
       description: Filter by source (hostname or IP address).
     - name: destination
       description: Filter by destination (hostname or IP address).
-    description: Display information about incidents (with filters).
+    description: Deprecated. Use GuardiCore v2 instead.
```

Comment:
This is a description of a command.
This description should go in the main integration description